azurelinuxagent/ga/exthandlers.py

```diff
@@ -67,7 +66,7 @@
 HANDLER_COMPLETE_NAME_PATTERN = re.compile(_HANDLER_PATTERN + r'$', re.IGNORECASE)
 HANDLER_PKG_EXT = ".zip"
 
-AGENT_STATUS_FILE = "waagent_status.json"
+AGENT_STATUS_FILE = "waagent_status.{0}.json"
```

Comment:
NIT: not necessary, but might consider naming the variable to include that this is a format string (i.e. can't be used before `.format`)

Reply:
Something like "AGENT_STATUS_FILE_FORMAT" ?